config/default.json

```diff
@@ -1,3 +1,8 @@
 {
-    "port": 3000
+    "port": 3000,
+    "debug": false,
+    "bot": {
+        "login": "MarmaBot",
+        "oauthToken": "bac80dd16db5599335f2b180d999b227a2470e45"
```

Comment:
Never commit tokens or other credentials. Please remove it from your Git history. Instead, fill it with an empty string, and create a `config/development.json` with the missing key (have a look on the [node-config](https://github.com/lorenwest/node-config) documentation for more details).